include/cantera/thermo/Elements.h

```diff
@@ -94,7 +96,19 @@ namespace Cantera
 //!     @param ename  String, Only the first 3 characters are significant
 //!     @return The atomic weight of the element
 //!     @exception CanteraError If a match is not found, throws a CanteraError
-double LookupWtElements(const std::string& ename);
+doublereal LookupWtElements(const std::string& ename);
+doublereal LookupWtElements(int atomicNumber);
```

Comment:
may as well just use `double` here. I think the `doublereal` alias is basically useless and avoid using it in any new code.